products/redis/api.yaml

```diff
@@ -172,3 +172,7 @@ objects:
           - :STANDARD_HA
         default_value: :BASIC
         input: true
+      - !ruby/object:Api::Type::UpdateMask
+        name: updateMask
+        description: The update mask (internal only)
+        output: true
```

Comment:
What is an output only update mask?

Reply:
I didn't know of a great way to describe this.

Do we place updateMask as a property or as a flag on a resource?

If it's a property, it's a property that is neither user-settable nor outputted  in any meaningful way.

Reply:
Right.  Personally, I think it's a flag on a resource, but I wasn't going to quibble about the representation - there's precedent for both.

Reply:
I'll make it a flag on a resource then.

I was worried about the case where an "updateMask" wasn't actually called "updateMask". In that case, we could add more flags to allow us to override the name...or we could just treat it as a property.

Reply:
drive-by thoughts:

* We probably don't need to futureproof- a name change will probably come with a behaviour change too, invalidating what we have anyways.

* If we do want to regardless, we could always make it a string-type field where the user provides what the field is called.